src/note/nullifier.rs

```diff
@@ -1,5 +1,7 @@
+use ff::Field;
```

Comment:
```suggestion
use group::Group;
```
Part of the suggested change below.